core/i18n/src/test/ITwinLocalization.test.ts

```diff
@@ -0,0 +1,152 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { assert } from "chai";
+import { Localization } from "@itwin/core-common";
+import { ITwinLocalization } from "../ITwinLocalization";
+
+describe("ITwinLocalization", () => {
+
+  let localization: Localization;
```

Comment:
Could move it closer to its first use, inside the getLocalizedString describe callback?

Reply:
There will be more tests in the future which use this - which is why it's defined outside of the \#getLocalizedString tests.  Granted, it's not used in the first set of tests (the \#initialize tests)... But once I add more tests, it will make more sense up top. I'll consider moving it down past the first set of tests when I add more tests though?

Reply:
Gotcha - no worries if it will all make sense :)